resources/web/wwi/animation.js

```diff
@@ -0,0 +1,204 @@
+/* global ResourceManager */
+'use strict';
+
+function Animation(url, scene, view, gui, loop) {
+  this.url = url;
+  this.scene = scene;
+  this.view = view;
+  this.gui = gui === undefined || gui === 'play' ? 'play' : 'pause';
+  this.loop = loop === undefined ? true : loop;
+  this.sliding = false;
+  this.onReady = null;
+};
+
+Animation.prototype = {
+  constructor: Animation,
+
+  init: function(onReady) {
+    var that = this;
+    this.onReady = onReady;
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.open('GET', this.url, true);
+    xmlhttp.overrideMimeType('application/json');
+    xmlhttp.onreadystatechange = function() {
+      if (xmlhttp.readyState === 4 && xmlhttp.status === 200)
+        that._setup(JSON.parse(xmlhttp.responseText));
+    };
+    xmlhttp.send();
+  },
+
+  moveSlider: function(event) {
+    if (!this.playSlider || !this.sliding)
+      return;
+
+    var w = event.target.clientWidth - 66; // size of the borders of the slider
+    var x = event.clientX - event.target.getBoundingClientRect().left - 48; // size of the left border (including play button) of the slider
+    var value = 100 * x / w;
+    if (value < 0)
+      value = 0;
+    else if (value >= 100)
+      value = 99.999;
```

Comment:
Hard coded value like this would deserve a comment.

Reply:
This is old code that I simply moved but not modified. I will check if I can understand why this was done.